pkg/apis/monitoring/v1/types.go

```diff
@@ -484,6 +484,8 @@ type StorageSpec struct {
 	Ephemeral *v1.EphemeralVolumeSource `json:"ephemeral,omitempty"`
 	// A PVC spec to be used by the Prometheus StatefulSets.
 	VolumeClaimTemplate EmbeddedPersistentVolumeClaim `json:"volumeClaimTemplate,omitempty"`
+	// Subpath for /prometheus volume mount
+	MountSubPath string `json:"name,omitempty" protobuf:"bytes,1,opt,name=mountSubPath"`
```

Comment:
```suggestion
	MountSubPath string `json:"mountSubPath,omitempty"`
```
Do we need the protobuf tag? We don't use it anywhere else in the CRDs.